helm/values.yaml

```diff
@@ -0,0 +1,34 @@
+psql:
+  jenkinsBuildId: "1"
+  jenkinsStartBuildDateTime: ""
+  jenkinsStartBuildUserName: ""
+  projectNamespace: ""
+  job:
+    backoffLimit: "0"
+    ttlSecondsAfterFinished: "10"
+    activeDeadlineSeconds: "1200"
+    container:
+      image:
+        repository: "docker-folio.dev.folio.org/org/folio/psql-dump-helm"
+        pullPolicy: Always
+        tag: ""
+      envVarsSecret: "s3-postgres-backups-credentials"
```

Comment:
can we rename the name to specify the purpose of secret map? Also please add comment which describes which values are expected to be defined in the map